packages/MarkdownEditor-Tests.package/MarkdownLineTest.class/instance/testPotentialBlockClass.st

```diff
@@ -0,0 +1,11 @@
+tests - block conversion
+testPotentialBlockClass
```

Comment:
`testPotentialBlockClasses`, we're not naming tests after their method name anymore ;)

Reply:
What would be a better name? I don't really want to list alle the Block Types that we are testing in the test name.

Reply:
The difference between current and proposed name is two characters small ^^

Reply:
Ahh. I'm blind I guess